### ISSUE TITLE
Error in the paired-heatmap 

### ISSUE THREAD

**Issue by helenamrusso** [biocore/mmvec#133]
Hi, I have recently started to use mmvec and I think it will be pretty useful for my project! 

I was able to run mmvec (I'm using Qiime2), generate the emperor biplot and the heatmap successfully, even though the heatmap was not showing the full table with the Taxon (maybe it was too big), and it got malformatted - I got something similar to what you specify [here](https://github.com/biocore/mmvec/pull/100#issuecomment-543422637).

However, for the paired-heatmap, I'm getting the error below:

(some FutureWarning: SparseDataFrame is deprecated and will be removed in a future version.)

 return self._constructor(new_data).__finalize__(self)
Traceback (most recent call last):
  File "/Users/helenarusso/opt/miniconda3/envs/qiime2-2020.2/lib/python3.6/site-packages/q2cli/commands.py", line 328, in __call__
    results = action(**arguments)
  File "</Users/helenarusso/opt/miniconda3/envs/qiime2-2020.2/lib/python3.6/site-packages/decorator.py:decorator-gen-484>", line 2, in paired_heatmap
  File "/Users/helenarusso/opt/miniconda3/envs/qiime2-2020.2/lib/python3.6/site-packages/qiime2/sdk/action.py", line 245, in bound_callable
    output_types, provenance)
  File "/Users/helenarusso/opt/miniconda3/envs/qiime2-2020.2/lib/python3.6/site-packages/qiime2/sdk/action.py", line 452, in _callable_executor_
    ret_val = self._callable(output_dir=temp_dir, **view_args)
  File "/Users/helenarusso/opt/miniconda3/envs/qiime2-2020.2/lib/python3.6/site-packages/mmvec/q2/_visualizers.py", line 72, in paired_heatmap
    color_palette)
  File "/Users/helenarusso/opt/miniconda3/envs/qiime2-2020.2/lib/python3.6/site-packages/mmvec/heatmap.py", line 202, in paired_heatmaps
    select_metabolites = metabolites_table[top_metabolites]
  File "/Users/helenarusso/opt/miniconda3/envs/qiime2-2020.2/lib/python3.6/site-packages/pandas/core/frame.py", line 3001, in __getitem__
    indexer = self.loc._convert_to_indexer(key, axis=1, raise_missing=True)
  File "/Users/helenarusso/opt/miniconda3/envs/qiime2-2020.2/lib/python3.6/site-packages/pandas/core/indexing.py", line 1285, in _convert_to_indexer
    return self._get_listlike_indexer(obj, axis, **kwargs)[1]
  File "/Users/helenarusso/opt/miniconda3/envs/qiime2-2020.2/lib/python3.6/site-packages/pandas/core/indexing.py", line 1092, in _get_listlike_indexer
    keyarr, indexer, o._get_axis_number(axis), raise_missing=raise_missing
  File "/Users/helenarusso/opt/miniconda3/envs/qiime2-2020.2/lib/python3.6/site-packages/pandas/core/indexing.py", line 1177, in _validate_read_indexer
    key=key, axis=self.obj._get_axis_name(axis)
**KeyError: "None of [Int64Index([   159,   3223,      7,    144,    226,    130,    475,   3625,\n             16342,  11914,\n            ...\n            226470, 220471, 221429, 223462, 210751, 228835, 179170,  99919,\n            214546, 231687],\n           dtype='int64', length=200)] are in the [columns]"**

(these numbers would be the metabolites features)

This is what I ran:
qiime mmvec paired-heatmap \
  --i-ranks ranks_LearningRate4.qza \
  --i-microbes-table microbes_mmvec.qza \
  --i-metabolites-table metabolites_mmvec.qza \
  --m-microbe-metadata-file metadata_microbes.txt \
  --m-microbe-metadata-column Taxon \
  --p-features Otu00003 \
  --p-features Otu00006 \
  --p-top-k-microbes 0 \
  --p-normalize rel_row \
  --p-top-k-metabolites 100 \
  --p-level 5 \
  --o-visualization ranks_LearningRate4_paired-heatmap-top2.qzv \
  --verbose

I converted the ranks.qza file to ranks.qzv (attached) and, indeed, the metabolites features are not in the columns - they are in the rows! The columns are the microbes Otus. Should it get transposed for this step? I have also tried to run it with a transposed metabolites table, I got the same error.

I also ran the cistic fibrosis example to check how it would work, and I didn't have any kind of problem like that - and the tables are all in the same format as mine (ranks.qzv file obtained has metabolites features in rows).

Thanks in advance!

<img width="1172" alt="ranks qzv file" src="https://user-images.githubusercontent.com/55901470/83537566-1a03e280-a4cb-11ea-8d7e-017e315aaf6b.png">

**Comment by mortonjt** [comment]
I bet that you got your biom tables transposed at some point.  The command `biom summarize table` can help you determine this.

**Comment by helenamrusso** [comment]
Thanks for the response!

I checked everything again, they are in the right format for both .biom files (columns represent the samples, the rows represent the features). In fact, it generated the ranks.qza file correctly: otus in columns, metabolites in rows. That is the same format for the ranks.qza I obtained running the cf example).

I don't really understand why it says that the metabolites features are not in columns. They need to be in rows in the ranks file, right?

**Comment by mortonjt** [comment]
There are two possibilities that could lead to that, namely
1. You got `--i-microbes-table microbes_mmvec.qza` and  `--i-metabolites-table metabolites_mmvec.qza` mixed up
2. You are using an out-dated version of mmvec.  We switched the orientation in the [version 1.0.1](https://github.com/biocore/mmvec/releases/tag/v1.0.1) release.

**Comment by helenamrusso** [comment]
Just a quick update, maybe it could be useful for Qiime2 and mmvec starters - as I sure am!

In the end, I think I have found out what was causing confusion. My metabolite features and my samples were only numbers - sometimes, even matching - and I think that caused confusion. I changed the sample names and features so they wouldn't match, ran exactly the same notebook and it now worked. 
The qza files were not mixed up, and my mmvec version is 1.0.4.

Thanks!
